packages/tools/client-debugger/client-debugger-chrome-extension/public/manifest.json

```diff
@@ -7,6 +7,10 @@
 		"default_icon": "images/Icon.png",
 		"default_popup": "popup/popup.html"
 	},
+	"browser_action": {
```

Comment:
We don't want to launch the devtools on button press. The devtools panel should be initialized when the user navigates to the appropriate tab in the browser devtools.

I don't think we want to change the actual initialization of the extension at all. Rather, we want to update the `DevtoolsView` component to add a timeout on its supported feature set query. We'll also want to display a notice noting that the Devtools were not found when the timeout is exceeded, and also have a button to search again (re-send the feature support query).

Reply:
I see. Updated.